lib/util/fs/index.ts

```diff
@@ -63,3 +66,17 @@ export async function ensureLocalDir(dirName): Promise<void> {
   const localDirName = join(localDir, dirName);
   await fs.ensureDir(localDirName);
 }
+
+export async function ensureCacheDir(
+  dirName,
+  envPathVar?: string
+): Promise<string> {
+  let envCacheDirName;
+  if (envPathVar) {
+    const env = getChildProcessEnv([envPathVar]);
```

Comment:
Does this mean it will only use the existing env if trustLevel=high?

Reply:
No, but `getChildProcessEnv` looks odd here anyway